src/Change.php

```diff
@@ -1128,7 +1128,7 @@ public static function showCentralList($start, $status = "process", $showgroupch
         $iterator = $DB->request($criteria);
 
         $total_row_count = count($iterator);
-        $displayed_row_count = (int)$_SESSION['glpidisplay_count_on_home'] > 0
+        $displayed_row_count = (int)$_SESSION['glpidisplay_count_on_home'] >= 0
```

Comment:
When `$_SESSION['glpidisplay_count_on_home']`, we still want to display the component to have the table head (i.e. the title/link/counter), without having table rows.

With proposed fix, nothing will be displayed.

Reply:
OK, I did not has that in mind